contracts/interfaces/common/IStakeable.sol

```diff
@@ -12,20 +12,24 @@ pragma solidity ^0.8.24;
 // Nexus: A suite of contracts for Modular Smart Account compliant with ERC-7579 and ERC-4337, developed by Biconomy.
 // Learn more at https://biconomy.io. For security issues, contact: security@biconomy.io
 
-/**
- * @title Stakeable Entity
- * @author Fil Makarov - <filipp.makarov@biconomy.io>
- */
+/// @title Stakeable Entity Interface
+/// @notice Interface for staking, unlocking, and withdrawing Ether on an EntryPoint.
+/// @dev Defines functions for managing stakes on an EntryPoint.
 interface IStakeable {
-    function addStake(
-        address epAddress,
-        uint32 unstakeDelaySec
-    ) external payable;
+    /// @notice Stakes a certain amount of Ether on an EntryPoint.
+    /// @dev The contract should have enough Ether to cover the stake.
+    /// @param epAddress Address of the EntryPoint where the stake is added.
+    /// @param unstakeDelaySec The delay in seconds before the stake can be unlocked.
+    function addStake(address epAddress, uint32 unstakeDelaySec) external payable;
 
+    /// @notice Unlocks the stake on an EntryPoint.
+    /// @dev This starts the unstaking delay after which funds can be withdrawn.
+    /// @param epAddress Address of the EntryPoint where the stake is unlocked.
```

Comment:
same here
you can copy this from StakeManager.sol of infinitism repo

Reply:
done